packages/@atjson/offset-annotations/src/annotations/ceros-embed.ts

```diff
@@ -4,6 +4,7 @@ export class CerosEmbed extends ObjectAnnotation<{
   url: string;
   aspectRatio: number;
   mobileAspectRatio?: number;
+  size: "small" | "medium" | "large";
```

Comment:
This needs to be optional
```suggestion
  size?: "small" | "medium" | "large";
```

Reply:
Updated code for mentioned comment